deploy-service/common/src/main/java/com/pinterest/deployservice/dao/AgentDAO.java

```diff
@@ -49,7 +49,7 @@
 
     void deleteAllById(String hostId) throws Exception;
 
-    // return how many agents are deploying for this env, regardless of deployId
+    // return how many agents are deploying and agent state is STOP (because agent with STOP state won't deploy in future) for this env, regardless of deployId
```

Comment:
can we change the comments to something like:

// return how many agents are deploying for this env, including agents whose state is STOP